CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributing to jupyter/accessibility
+
+## Code of Conduct
+
+This project follows the [Jupyter Code of Conduct].
+
+## Goal
+
+> To facilitate improving the accessibility of interactive computing, we are trying to make it easy to:
+>
+> - capture the state of an accessibility feature across number of separate Jupyter
+>   repositories at different states of development as [pull requests]
+> - deploy them _together_ on [Binder] to share with people for testing
+> - assess them on [GitHub Actions] to provide reports, raw data, and feedback
+> - work as a team to get the pull requests merged
+> - show progress towards an accessibility _Roadmap_
+
+## Core Workflow
+
+1. get a free GitHub account
+2. make one (or more) [pull requests] to one (or more) Jupyter repositories
+3. make a pull request to `jupyter/accessibility` that changes `repos.toml` to
```

Comment:
Add text and link specifying _which_ branch to target, e.g.

```
make a pull request targeting the [`ci` branch][ci-branch]

[ci-branch]: https://github.com/jupyter/accessibility/compare<whatever>
```

Reply:
updated... i couldn't find a _nice_ way to make a good, inviting link target; future work would include a little builder like the nbgitpuller or binder link.